README.md

```diff
@@ -32,6 +32,14 @@ A client represents a connection to the Recurly API. Every call to the server ex
 client := recurly.NewClient("<apikey>")
 ```
 
+To access Recurly servers in Europe you have to initialize the client with the argument options and set region with the value "EU" using the struct ClientOptions.
+
+```go
+client := recurly.NewClientWithOptions("<apikey>", &ClientOptions{
+    region: EU
```

Comment:
Since developers using this package will be calling it from outside the package, they will need to include the `recurly` package name:

```suggestion
client := recurly.NewClientWithOptions("<apikey>", &recurly.ClientOptions{
    Region: recurly.EU
```